charts/agent/templates/_helpers.tpl

```diff
@@ -319,6 +319,49 @@ Use global sysdig tags for agent
     {{- end -}}
 {{- end -}}
 
+{{/*
+Determine the plan settings (monitor/secure) set in the sysdig-deploy chart
+and set the agent chart parameters accordingly
+*/}}
+{{- define "agent.planSettings" -}}
+    {{/* Determine if secure or secure_light mode is being requested in Agent settings and store state */}}
+    {{- if hasKey .Values.sysdig.settings "feature" }}
+        {{- $_ := set .Values "secureLight" (eq .Values.sysdig.settings.feature.mode "secure_light") }}
+        {{- $_ := set .Values "secureFeatProvided" (or .Values.secureLight
```

Comment:
Why are we storing those values? Can't we just have them as variables since they're used only here?